### PR TITLE
DeriveFromPublicKey with network

### PR DESCRIPTION
**Pull request by qiwu7**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -9,5 +9,8 @@ require (
 	github.com/filecoin-project/lotus v0.7.1
 	github.com/filecoin-project/specs-actors v0.9.10
 	github.com/minio/blake2b-simd v0.0.0-20160723061019-3f5f724cb5b1
+	github.com/stretchr/testify v1.6.1
 	github.com/whyrusleeping/cbor-gen v0.0.0-20200826160007-0b9f6c5fb163
 )
+
+replace github.com/filecoin-project/go-address => ../go-address
```
Comment (qiwu7) [inline go.mod:16]: waiting on this [PR](https://github.com/filecoin-project/go-address/pull/14) to land before removing

Comment (jleni) [reply go.mod:16]: great!

**Comment by qiwu7** [comment]
@jleni updated! Please take a look.

**Comment by jleni** [comment]
ok, I improved things so we can get the correct error message. The reason some tests fail is because this PR is not receiving CI secrets

File: go.mod (hunk 1 above)
Comment (fublok77) [reply go.mod]: Claim my asset

**Pull request merged**